src/main/java/com/mathworks/ci/MatlabBuilder.java

```diff
@@ -101,6 +102,14 @@ private String getCustomMatlabCommand() {
     private void setEnv(EnvVars env) {
         this.env = env;
     }
+    
+    private void setFileSeparator(String fileSeparator) {
+        this.fileSeparator = fileSeparator;
+    }
+    
+    private String getFileSeparator() {
+        return this.fileSeparator;
+    }
```

Comment:
Getters/setters for private variables seem unnecessary. Why not just access the private field directly?